modules/pool/pool.gql

```diff
@@ -165,7 +165,19 @@ type GqlPoolMinimal {
     """
     The wallet address of the owner of the pool. Pool owners can set certain properties like swapFees or AMP.
     """
-    owner: Bytes
+    owner: Bytes @deprecated(reason: "Use swapFeeManager instead")
+    """
+    Account empowered to set static swap fees for a pool (when 0 on V2 swap fees are immutable, on V3 delegate to governance)
+    """
+    swapFeeManager: Bytes
```

Comment:
not sure why this is bytes, lets make it string

Reply:
I guess Bytes naming comes from the contracts and it's meant to represent 0x strings. In TS it's just a String alias like BigDecimal is:
https://github.com/balancer/backend/blob/pool-roles/codegen.ts/#L15-L16
Maybe we should remove the mappings altogether to make it more understandable in the docs?

Reply:
agree, lets get rid of bytes and just use strings

Reply:
leave the bigdecimals for now, only do strings